src/ducks/transactions/TransactionModal/TransactionModalInfoContent.jsx

```diff
@@ -196,21 +196,22 @@ const TransactionModalInfoContent = props => {
 
       <DeleteTransactionRow transaction={transaction} onDelete={requestClose} />
 
-      {selectedRow == 'category' ? (
+      {selectedRow == 'category' && (
```

Comment:
nitpick but we I think we should always use strict equality. Maybe we should update it in our eslint config.